src/inc/quic_datapath.h

```diff
@@ -578,6 +577,9 @@ typedef struct CXPLAT_UDP_CONFIG {
     uint8_t CibirIdOffsetSrc;           // CIBIR ID offset in source CID
     uint8_t CibirIdOffsetDst;           // CIBIR ID offset in destination CID
     uint8_t CibirId[6];                 // CIBIR ID data
+
+    uint8_t UseQTIP;
+    uint8_t OverrideGlobalQTIPSettings;
```

Comment:
These can/should be `Flags`. I'm also not sure (yet) why you need two...

Reply:
resolved.